server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,22 +290,63 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
+
+#console-action-bar {
+    min-height: 20px;
+    padding: 10px;
+
+    background-color: #333;
+
+    div {
+        float: right;
+    }
+
+    a {
+        padding-right: 15px;
+        color: yellow;
+    }
+
+    &.white-theme {
+        background-color: white;
+        a {
+            color: #1F66BD;
+        }
+    }
+}
+
 #buildoutput_pre {
     overflow: auto;
+
+    white-space: pre;
     white-space: pre-wrap;
+    word-break: break-all;
+    word-wrap: break-word;
 
     padding-left: 10px;
     padding-bottom: 10px;
 
-    @each $color in 'black', 'red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white', 'gray' {
+
+    background-color: #333;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces